db/migrate/20170421153056_add_email_count_to_signatures.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailCountToSignatures < ActiveRecord::Migration
+  def change
+    add_column :signatures, :email_count, :integer
```

Comment:
@h-lame yes, adding 0 as the default would lock the whole table during migration whilst it backfills 30 million rows. However since we're emptying the whole table shortly that would be a good time to add it.